sql/parser/builtins.go

```diff
@@ -1300,7 +1300,8 @@ var uniqueBytesState struct {
 	nanos uint64
 }
 
-func generateUniqueBytes(nodeID uint32) DBytes {
+// GenerateUniqueBytes generates unique bytes for a unique id.
```

Comment:
Now that this is exported the comment should explain what properties the generated value has (i.e. why isn't it a random UUID? It looks like the major goal is a rough sortability, which doesn't matter for this change so I'm not sure this is the right function to use here). We should also clean up the interface: return `[]byte` instead of the sql-specific `DBytes` and the argument should probably be a `roachpb.NodeID`.